errors/index.js

```diff
@@ -1,22 +1,20 @@
-
-exports.handleCustomErrors = (err, req,res,next) => {
-    if(err.status && err.msg) {
-        res.status(err.status).send({msg: err.msg}); 
-    } else {
-        next(err);
-    }
-}
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
 
 exports.handlePsqlErrors = (err, req, res, next) => {
-    if (err.code === '22P02'){
-        res.status(400).send({msg: 'Bad request'})
-    } else {
-        next(err);
-    }
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Bad request" });
+  } else {
+    next(err);
+  }
 };
```

Comment:
Again, good work and specifically dealing with the PSQL error for an invalid data type.